### PR TITLE
adding vcl code and stale-if-error/while-validate cache control headers

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: deploy.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ function doUpload(files, dest, bucket, region){
 		destination : dest,
 		bucket : bucket,
 		strip : '1',
-		cacheControl : 'public, max-age=30'
+		cacheControl : 'public, max-age=30, stale-while-revalidate=30, stale-if-error=43200'
```
Comment (matthew-andrews) [inline deploy.js]: stale while revalidate leads to pretty crazy behaviour.  30s should be ok though :)

Comment (matthew-andrews) [inline deploy.js]: also don't we need some special VCL to make this stale magic happen?
https://docs.fastly.com/guides/performance-tuning/serving-stale-content.html

**Comment by matthew-andrews** [comment]
:+1:

File: deploy.js (hunk 1 above)
Comment (aintgoin2goa) [inline deploy.js]: apparently we do - have added it

**Pull request merged**